examples/image_classification/README.md

```diff
@@ -0,0 +1,393 @@
+var goldenDataset = dataset.subList(0, 15);
+var taskDataset = dataset.subList(15, dataset.size());
+```
+
+Function `readTSV` takes file name and returns `List<Map<String, String>>`, where each map keys are colunm names in file 
+and values are values in corresponding row.
+
+Create control tasks. In small pools, control tasks should account for 10–20% of all tasks.
+
+**Tip**: Make sure to include different variations of correct responses in equal amounts.
+
+```java
+var goldenTasks = goldenDataset.stream().map(row -> {
+        var task = new Task(
+        poolId,
+        Map.of("image", row.get("url")),
+        List.of(new KnownSolution(Map.of("result", row.get("label")), 0.0))
+        );
+        task.setInfiniteOverlap(true);
+        return task;
+        }).collect(Collectors.toUnmodifiableList());
+```
+
+Create pool tasks.
+
+```java
+var tasks = taskDataset.stream().map(row -> {
+        var task = new Task(poolId, Map.of("image", row.get("url")));
+        task.setOverlap(3);
+        return task;
+        }).collect(Collectors.toUnmodifiableList());
+```
+
+Upload tasks.
+
+```java
+var taskCreateRequestParameters = new TaskCreateRequestParameters();
+taskCreateRequestParameters.setAllowDefaults(true);
+clientFactory.getTaskClient().createTasks(tasks, taskCreateRequestParameters);
+clientFactory.getTaskClient().createTasks(goldenTasks, taskCreateRequestParameters);
+```
+
+Start the pool.
+
+**Important.** Remember that real Toloka performers will complete the tasks. Double check that everything is correct 
+with your project configuration before you start the pool.
+
+```java
+clientFactory.getPoolClient().openPool(poolId);
+```
+
+## Receiving responses
+
+Wait until the pool is completed.
+
+```java
+Pool pool = clientFactory.getPoolClient().getPool(poolId);
+while (pool.getStatus() != PoolStatus.CLOSED) {
+    try {
+        Thread.sleep(1000 * 60);
+    } catch (InterruptedException ignored) {
+    }
+    pool = clientFactory.getPoolClient().getPool(poolId);
+}
+```
+
+Get responses.
+
+When all the tasks are completed, look at the responses from performers.
+
+Aggregate results using the Dawid-Skene model.
+
+```java
+PoolAggregatedSolutionRequest request = new DawidSkenePoolAggregatedSolutionRequest(poolId, List.of(new DawidSkenePoolAggregatedSolutionRequest.Field("result")));
+AggregatedSolutionOperation result = clientFactory.getAggregatedSolutionClient().aggregateSolutionsByPool(request).waitToComplete();
+var solutionSearchRequest = AggregatedSolutionSearchRequest.make().limit(250).done();
+```
+Get list of tasks with aggregated results.
+
+```java
+var aggregatedSolutionSearchResult = clientFactory
+                                            .getAggregatedSolutionClient()
+                                            .findAggregatedSolutions(result.getId(), solutionSearchRequest);
+var aggregatedSolutions = aggregatedSolutionSearchResult.getItems();
+```
+
+Aggregated answer can be founded in `outputValues`.
```

Comment:
We need to demonstrate final result to user

Reply:
To get final result user need to join aggregated solutiona and all tasks from the pool by `taskId`. Variant to do it is presented in `ImageClassification.java`. Add information about it in `README.md`